src/script/microbit-interfacing/Microbits.ts

```diff
@@ -30,20 +30,32 @@ export enum HexOrigin {
   PROPRIETARY,
 }
 
+type HexType = 'bluetooth' | 'radio-sender' | 'radio-bridge' | 'radio-local';
+
 type UARTMessageType = 'g' | 's';
 
+export const getHexFileUrl = (version: 1 | 2 | 'universal', type: HexType) => {
+  if (type === 'bluetooth') {
+    return {
+      1: 'firmware/ml-microbit-cpp-version-combined.hex',
+      2: 'firmware/MICROBIT.hex',
+      universal: 'firmware/universal-hex.hex',
+    }[version];
+  }
+  if (version !== 2) {
+    throw new Error('Only V2 is supported');
```

Comment:
This can presumably happen and we should communicate it somehow. I'll raise a ticket for discussion.

Reply:
https://microbit-global.monday.com/boards/1356069004/views/10076885/pulses/1369901005